### PR TITLE
Add TomEE 8.0.0-M3 on Java 11

### PR DESCRIPTION
**Pull request by jgallimore**

### CODE REVIEW

File: library/tomee [hunk 1]
```diff
@@ -55,5 +54,17 @@ Directory: 8-jre-8.0.0-M2-plus
 Tags: 8-jre-8.0.0-M2-microprofile
 Directory: 8-jre-8.0.0-M2-microprofile
 
-Tags: 8-jre-8.0.0-M2-webprofile, latest
+Tags: 8-jre-8.0.0-M2-webprofile
```
Comment (otaviojava) [inline library/tomee]: Please, update this tag to `8-jre-8.0.0-M3-webprofile`

File: library/tomee [hunk 2]
```diff
@@ -55,5 +54,17 @@ Directory: 8-jre-8.0.0-M2-plus
...
 Directory: 8-jre-8.0.0-M2-webprofile
```
Comment (otaviojava) [inline library/tomee]: the 8.0.0-M2-webprofile directory does not exist anymore, please, update it to M3.

**Comment by otaviojava** [comment]
Hey @jgallimore 
I found the issue:

https://github.com/tomitribe/docker-tomee/blob/master/11-jre-8.0.0-M3-microprofile/Dockerfile#L1

There is not this option to Java 11:
https://hub.docker.com/_/openjdk/

Try this option: 11-jdk

FROM openjdk:11-jdk

**Comment by jgallimore** [comment]
I just spotted that too. Updating and trying again. Thanks for your help!

**Comment by otaviojava** [comment]
Done :)

**Comment by jgallimore** [comment]
It looks happy now, thanks for your help @otaviojava !

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 5afe821..717a06d 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,126 +1,18 @@
 tomee:8-jre-1.7.5-jaxrs @ amd64
-tomee:8-jre-1.7.5-jaxrs @ arm32v5
-tomee:8-jre-1.7.5-jaxrs @ arm32v7
-tomee:8-jre-1.7.5-jaxrs @ arm64v8
-tomee:8-jre-1.7.5-jaxrs @ i386
-tomee:8-jre-1.7.5-jaxrs @ ppc64le
-tomee:8-jre-1.7.5-jaxrs @ s390x
 tomee:8-jre-1.7.5-plume @ amd64
-tomee:8-jre-1.7.5-plume @ arm32v5
-tomee:8-jre-1.7.5-plume @ arm32v7
-tomee:8-jre-1.7.5-plume @ arm64v8
-tomee:8-jre-1.7.5-plume @ i386
-tomee:8-jre-1.7.5-plume @ ppc64le
-tomee:8-jre-1.7.5-plume @ s390x
 tomee:8-jre-1.7.5-plus @ amd64
-tomee:8-jre-1.7.5-plus @ arm32v5
-tomee:8-jre-1.7.5-plus @ arm32v7
-tomee:8-jre-1.7.5-plus @ arm64v8
-tomee:8-jre-1.7.5-plus @ i386
-tomee:8-jre-1.7.5-plus @ ppc64le
-tomee:8-jre-1.7.5-plus @ s390x
 tomee:8-jre-1.7.5-webprofile @ amd64
-tomee:8-jre-1.7.5-webprofile @ arm32v5
-tomee:8-jre-1.7.5-webprofile @ arm32v7
-tomee:8-jre-1.7.5-webprofile @ arm64v8
-tomee:8-jre-1.7.5-webprofile @ i386
-tomee:8-jre-1.7.5-webprofile @ ppc64le
-tomee:8-jre-1.7.5-webprofile @ s390x
 tomee:8-jre-7.0.4-plume @ amd64
-tomee:8-jre-7.0.4-plume @ arm32v5
-tomee:8-jre-7.0.4-plume @ arm32v7
-tomee:8-jre-7.0.4-plume @ arm64v8
-tomee:8-jre-7.0.4-plume @ i386
-tomee:8-jre-7.0.4-plume @ ppc64le
-tomee:8-jre-7.0.4-plume @ s390x
 tomee:8-jre-7.0.4-plus @ amd64
-tomee:8-jre-7.0.4-plus @ arm32v5
-tomee:8-jre-7.0.4-plus @ arm32v7
-tomee:8-jre-7.0.4-plus @ arm64v8
-tomee:8-jre-7.0.4-plus @ i386
-tomee:8-jre-7.0.4-plus @ ppc64le
-tomee:8-jre-7.0.4-plus @ s390x
 tomee:8-jre-7.0.4-webprofile @ amd64
-tomee:8-jre-7.0.4-webprofile @ arm32v5
-tomee:8-jre-7.0.4-webprofile @ arm32v7
-tomee:8-jre-7.0.4-webprofile @ arm64v8
-tomee:8-jre-7.0.4-webprofile @ i386
-tomee:8-jre-7.0.4-webprofile @ ppc64le
-tomee:8-jre-7.0.4-webprofile @ s390x
 tomee:8-jre-7.0.5-plume @ amd64
-tomee:8-jre-7.0.5-plume @ arm32v5
-tomee:8-jre-7.0.5-plume @ arm32v7
-tomee:8-jre-7.0.5-plume @ arm64v8
-tomee:8-jre-7.0.5-plume @ i386
-tomee:8-jre-7.0.5-plume @ ppc64le
-tomee:8-jre-7.0.5-plume @ s390x
 tomee:8-jre-7.0.5-plus @ amd64
-tomee:8-jre-7.0.5-plus @ arm32v5
-tomee:8-jre-7.0.5-plus @ arm32v7
-tomee:8-jre-7.0.5-plus @ arm64v8
-tomee:8-jre-7.0.5-plus @ i386
-tomee:8-jre-7.0.5-plus @ ppc64le
-tomee:8-jre-7.0.5-plus @ s390x
 tomee:8-jre-7.0.5-webprofile @ amd64
-tomee:8-jre-7.0.5-webprofile @ arm32v5
-tomee:8-jre-7.0.5-webprofile @ arm32v7
-tomee:8-jre-7.0.5-webprofile @ arm64v8
-tomee:8-jre-7.0.5-webprofile @ i386
-tomee:8-jre-7.0.5-webprofile @ ppc64le
-tomee:8-jre-7.0.5-webprofile @ s390x
 tomee:8-jre-7.1.0-microprofile @ amd64
-tomee:8-jre-7.1.0-microprofile @ arm32v5
-tomee:8-jre-7.1.0-microprofile @ arm32v7
-tomee:8-jre-7.1.0-microprofile @ arm64v8
-tomee:8-jre-7.1.0-microprofile @ i386
-tomee:8-jre-7.1.0-microprofile @ ppc64le
-tomee:8-jre-7.1.0-microprofile @ s390x
 tomee:8-jre-7.1.0-plume @ amd64
-tomee:8-jre-7.1.0-plume @ arm32v5
-tomee:8-jre-7.1.0-plume @ arm32v7
-tomee:8-jre-7.1.0-plume @ arm64v8
-tomee:8-jre-7.1.0-plume @ i386
-tomee:8-jre-7.1.0-plume @ ppc64le
-tomee:8-jre-7.1.0-plume @ s390x
 tomee:8-jre-7.1.0-plus @ amd64
-tomee:8-jre-7.1.0-plus @ arm32v5
-tomee:8-jre-7.1.0-plus @ arm32v7
-tomee:8-jre-7.1.0-plus @ arm64v8
-tomee:8-jre-7.1.0-plus @ i386
-tomee:8-jre-7.1.0-plus @ ppc64le
-tomee:8-jre-7.1.0-plus @ s390x
 tomee:8-jre-7.1.0-webprofile @ amd64
-tomee:8-jre-7.1.0-webprofile @ arm32v5
-tomee:8-jre-7.1.0-webprofile @ arm32v7
-tomee:8-jre-7.1.0-webprofile @ arm64v8
-tomee:8-jre-7.1.0-webprofile @ i386
-tomee:8-jre-7.1.0-webprofile @ ppc64le
-tomee:8-jre-7.1.0-webprofile @ s390x
-tomee:8-jre-8.0.0-M2-microprofile @ amd64
-tomee:8-jre-8.0.0-M2-microprofile @ arm32v5
-tomee:8-jre-8.0.0-M2-microprofile @ arm32v7
-tomee:8-jre-8.0.0-M2-microprofile @ arm64v8
-tomee:8-jre-8.0.0-M2-microprofile @ i386
-tomee:8-jre-8.0.0-M2-microprofile @ ppc64le
-tomee:8-jre-8.0.0-M2-microprofile @ s390x
-tomee:8-jre-8.0.0-M2-plume @ amd64
-tomee:8-jre-8.0.0-M2-plume @ arm32v5
-tomee:8-jre-8.0.0-M2-plume @ arm32v7
-tomee:8-jre-8.0.0-M2-plume @ arm64v8
-tomee:8-jre-8.0.0-M2-plume @ i386
-tomee:8-jre-8.0.0-M2-plume @ ppc64le
-tomee:8-jre-8.0.0-M2-plume @ s390x
-tomee:8-jre-8.0.0-M2-plus @ amd64
-tomee:8-jre-8.0.0-M2-plus @ arm32v5
-tomee:8-jre-8.0.0-M2-plus @ arm32v7
-tomee:8-jre-8.0.0-M2-plus @ arm64v8
-tomee:8-jre-8.0.0-M2-plus @ i386
-tomee:8-jre-8.0.0-M2-plus @ ppc64le
-tomee:8-jre-8.0.0-M2-plus @ s390x
+tomee:11-jre-8.0.0-M3-microprofile @ amd64
+tomee:11-jre-8.0.0-M3-plume @ amd64
+tomee:11-jre-8.0.0-M3-plus @ amd64
 tomee:latest @ amd64
-tomee:latest @ arm32v5
-tomee:latest @ arm32v7
-tomee:latest @ arm64v8
-tomee:latest @ i386
-tomee:latest @ ppc64le
-tomee:latest @ s390x
diff --git a/_bashbrew-list b/_bashbrew-list
index a01f119..1eb9916 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -12,8 +12,8 @@ tomee:8-jre-7.1.0-microprofile
 tomee:8-jre-7.1.0-plume
 tomee:8-jre-7.1.0-plus
 tomee:8-jre-7.1.0-webprofile
-tomee:8-jre-8.0.0-M2-microprofile
-tomee:8-jre-8.0.0-M2-plume
-tomee:8-jre-8.0.0-M2-plus
-tomee:8-jre-8.0.0-M2-webprofile
+tomee:11-jre-8.0.0-M3-microprofile
+tomee:11-jre-8.0.0-M3-plume
+tomee:11-jre-8.0.0-M3-plus
+tomee:11-jre-8.0.0-M3-webprofile
 tomee:latest
diff --git a/tomee_8-jre-8.0.0-M2-microprofile/Dockerfile b/tomee_11-jre-8.0.0-M3-microprofile/Dockerfile
similarity index 79%
rename from tomee_8-jre-8.0.0-M2-microprofile/Dockerfile
rename to tomee_11-jre-8.0.0-M3-microprofile/Dockerfile
index 86d4cb8..344f84a 100644
--- a/tomee_8-jre-8.0.0-M2-microprofile/Dockerfile
+++ b/tomee_11-jre-8.0.0-M3-microprofile/Dockerfile
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM openjdk:11.0-jdk
 
 ENV PATH /usr/local/tomee/bin:$PATH
 RUN mkdir -p /usr/local/tomee
@@ -17,7 +17,8 @@ ENV GPG_KEYS \
     DBCCD103B8B24F86FFAAB025C8BB472CD297D428 \
     F067B8140F5DD80E1D3B5D92318242FE9A0B1183 \
     FAA603D58B1BA4EDF65896D0ED340E0E6D545F97 \
-    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1
+    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1 \
+    294A395FFDC9FCF25A7E2BFDCF6FC99C2CC77782
 
 RUN set -xe \
     && for key in $GPG_KEYS; do \
@@ -27,12 +28,12 @@ RUN set -xe \
     done
 
 RUN set -x \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-microprofile.tar.gz.asc -o tomee.tar.gz.asc \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-microprofile.tar.gz -o tomee.tar.gz \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-microprofile.tar.gz.asc -o tomee.tar.gz.asc \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-microprofile.tar.gz -o tomee.tar.gz \
 	&& gpg --batch --verify tomee.tar.gz.asc tomee.tar.gz \
 	&& tar -zxf tomee.tar.gz \
-	&& mv apache-tomee-microprofile-8.0.0-M2/* /usr/local/tomee \
-	&& rm -Rf apache-tomee-microprofile-8.0.0-M2 \
+	&& mv apache-tomee-microprofile-8.0.0-M3/* /usr/local/tomee \
+	&& rm -Rf apache-tomee-microprofile-8.0.0-M3 \
 	&& rm bin/*.bat \
 	&& rm tomee.tar.gz*
 
diff --git a/tomee_8-jre-8.0.0-M2-plume/Dockerfile b/tomee_11-jre-8.0.0-M3-plume/Dockerfile
similarity index 79%
rename from tomee_8-jre-8.0.0-M2-plume/Dockerfile
rename to tomee_11-jre-8.0.0-M3-plume/Dockerfile
index ed5a1c9..9625925 100644
--- a/tomee_8-jre-8.0.0-M2-plume/Dockerfile
+++ b/tomee_11-jre-8.0.0-M3-plume/Dockerfile
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM openjdk:11.0-jdk
 
 ENV PATH /usr/local/tomee/bin:$PATH
 RUN mkdir -p /usr/local/tomee
@@ -17,7 +17,8 @@ ENV GPG_KEYS \
     DBCCD103B8B24F86FFAAB025C8BB472CD297D428 \
     F067B8140F5DD80E1D3B5D92318242FE9A0B1183 \
     FAA603D58B1BA4EDF65896D0ED340E0E6D545F97 \
-    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1
+    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1 \
+    294A395FFDC9FCF25A7E2BFDCF6FC99C2CC77782
 
 RUN set -xe \
     && for key in $GPG_KEYS; do \
@@ -27,12 +28,12 @@ RUN set -xe \
     done
 
 RUN set -x \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-plume.tar.gz.asc -o tomee.tar.gz.asc \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-plume.tar.gz -o tomee.tar.gz \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-plume.tar.gz.asc -o tomee.tar.gz.asc \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-plume.tar.gz -o tomee.tar.gz \
 	&& gpg --batch --verify tomee.tar.gz.asc tomee.tar.gz \
 	&& tar -zxf tomee.tar.gz \
-	&& mv apache-tomee-plume-8.0.0-M2/* /usr/local/tomee \
-	&& rm -Rf apache-tomee-plume-8.0.0-M2 \
+	&& mv apache-tomee-plume-8.0.0-M3/* /usr/local/tomee \
+	&& rm -Rf apache-tomee-plume-8.0.0-M3 \
 	&& rm bin/*.bat \
 	&& rm tomee.tar.gz*
 
diff --git a/tomee_8-jre-8.0.0-M2-plus/Dockerfile b/tomee_11-jre-8.0.0-M3-plus/Dockerfile
similarity index 80%
rename from tomee_8-jre-8.0.0-M2-plus/Dockerfile
rename to tomee_11-jre-8.0.0-M3-plus/Dockerfile
index b6a89da..2a6d925 100644
--- a/tomee_8-jre-8.0.0-M2-plus/Dockerfile
+++ b/tomee_11-jre-8.0.0-M3-plus/Dockerfile
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM openjdk:11.0-jdk
 
 ENV PATH /usr/local/tomee/bin:$PATH
 RUN mkdir -p /usr/local/tomee
@@ -17,7 +17,8 @@ ENV GPG_KEYS \
     DBCCD103B8B24F86FFAAB025C8BB472CD297D428 \
     F067B8140F5DD80E1D3B5D92318242FE9A0B1183 \
     FAA603D58B1BA4EDF65896D0ED340E0E6D545F97 \
-    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1
+    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1 \
+    294A395FFDC9FCF25A7E2BFDCF6FC99C2CC77782
 
 RUN set -xe \
     && for key in $GPG_KEYS; do \
@@ -27,12 +28,12 @@ RUN set -xe \
     done
 
 RUN set -x \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-plus.tar.gz.asc -o tomee.tar.gz.asc \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-plus.tar.gz -o tomee.tar.gz \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-plus.tar.gz.asc -o tomee.tar.gz.asc \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-plus.tar.gz -o tomee.tar.gz \
 	&& gpg --batch --verify tomee.tar.gz.asc tomee.tar.gz \
 	&& tar -zxf tomee.tar.gz \
-	&& mv apache-tomee-plus-8.0.0-M2/* /usr/local/tomee \
-	&& rm -Rf apache-tomee-plus-8.0.0-M2 \
+	&& mv apache-tomee-plus-8.0.0-M3/* /usr/local/tomee \
+	&& rm -Rf apache-tomee-plus-8.0.0-M3 \
 	&& rm bin/*.bat \
 	&& rm tomee.tar.gz*
 
diff --git a/tomee_latest/Dockerfile b/tomee_latest/Dockerfile
index fa793cb..19cfe36 100644
--- a/tomee_latest/Dockerfile
+++ b/tomee_latest/Dockerfile
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre
+FROM openjdk:11.0-jdk
 
 ENV PATH /usr/local/tomee/bin:$PATH
 RUN mkdir -p /usr/local/tomee
@@ -17,7 +17,8 @@ ENV GPG_KEYS \
     DBCCD103B8B24F86FFAAB025C8BB472CD297D428 \
     F067B8140F5DD80E1D3B5D92318242FE9A0B1183 \
     FAA603D58B1BA4EDF65896D0ED340E0E6D545F97 \
-    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1
+    C92604B0DEC5C62CFF5801E73D4683C24EDC64D1 \
+    294A395FFDC9FCF25A7E2BFDCF6FC99C2CC77782
 
 RUN set -xe \
     && for key in $GPG_KEYS; do \
@@ -27,12 +28,12 @@ RUN set -xe \
     done
 
 RUN set -x \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-webprofile.tar.gz.asc -o tomee.tar.gz.asc \
-	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M2/apache-tomee-8.0.0-M2-webprofile.tar.gz -o tomee.tar.gz \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-webprofile.tar.gz.asc -o tomee.tar.gz.asc \
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-8.0.0-M3/apache-tomee-8.0.0-M3-webprofile.tar.gz -o tomee.tar.gz \
 	&& gpg --batch --verify tomee.tar.gz.asc tomee.tar.gz \
 	&& tar -zxf tomee.tar.gz \
-	&& mv apache-tomee-webprofile-8.0.0-M2/* /usr/local/tomee \
-	&& rm -Rf apache-tomee-webprofile-8.0.0-M2 \
+	&& mv apache-tomee-webprofile-8.0.0-M3/* /usr/local/tomee \
+	&& rm -Rf apache-tomee-webprofile-8.0.0-M3 \
 	&& rm bin/*.bat \
 	&& rm tomee.tar.gz*
 
```

</details>

**Comment by yosifkit** [comment]
Did you still want to tag the images as `jre` even though they now have the JDK? Up to you either way.

**Comment by otaviojava** [comment]
Hey @yosifkit 
can we keep with jre, then create a second PR with this update?

**Comment by j4fm** [comment]
The image is tagged with jre so +1 for putting the jre base image back in

**Comment by jgallimore** [comment]
When we used openjdk:11-jre, this PR failed with:

 - tomee:11-jre-8.0.0-M3-microprofile (FROM openjdk:11-jre) -- completely
unsupported base!
 - tomee:11-jre-8.0.0-M3-plume (FROM openjdk:11-jre) -- completely
unsupported base!
 - tomee:11-jre-8.0.0-M3-plus (FROM openjdk:11-jre) -- completely
unsupported base!

So - which base image can we use to achieve this?
https://hub.docker.com/_/openjdk/ - I could be reading this wrong, but it
looks like all the supported tags point to a JDK.

Jon

**Comment by tianon** [comment]
Correct, the new releases do not include an official JRE for OpenJDK 11 (https://github.com/docker-library/openjdk/issues/320#issuecomment-494417274 -- "JDK 11 has no official JRE build.").

That being said, the new AdoptOpenJDK images (https://hub.docker.com/_/adoptopenjdk) *do* include a Java 11 JRE variant if you wanted to consider switching the base entirely.  Up to you -- we're happy to merge this as-is and let you consider that separately if you'd like.

**Comment by tianon** [comment]
Given https://github.com/docker-library/official-images/pull/6041#issuecomment-499764173, I'm going to merge this as-is and let you figure out your future plans separately. :+1:

Build test of #6041; 1cbc7e40; `amd64` (`tomee`):

```console

$ bashbrew build tomee:8-jre-1.7.5-jaxrs
Using bashbrew/cache:0de3f263d8f9f78187faa33eef9e8a7997e794357438b942f9a46e5ac1930b62 (tomee:8-jre-1.7.5-jaxrs)
Tagging tomee:8-jre-1.7.5-jaxrs

$ test/run.sh tomee:8-jre-1.7.5-jaxrs
testing tomee:8-jre-1.7.5-jaxrs
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-1.7.5-plume
Using bashbrew/cache:440ff20586528d1597b134dec9317ed3816dbb10ec3ac962ec84c9c7ff9de998 (tomee:8-jre-1.7.5-plume)
Tagging tomee:8-jre-1.7.5-plume

$ test/run.sh tomee:8-jre-1.7.5-plume
testing tomee:8-jre-1.7.5-plume
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-1.7.5-plus
Using bashbrew/cache:6bb8a0d0a4edc4ed97824dda0471c4922a9897ca0f744866611632e334b1eb6a (tomee:8-jre-1.7.5-plus)
Tagging tomee:8-jre-1.7.5-plus

$ test/run.sh tomee:8-jre-1.7.5-plus
testing tomee:8-jre-1.7.5-plus
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-1.7.5-webprofile
Using bashbrew/cache:0db69888f29492d5a4a9c596fe8d6c3966a50cd0d915fa755f1c22ac002c09b3 (tomee:8-jre-1.7.5-webprofile)
Tagging tomee:8-jre-1.7.5-webprofile

$ test/run.sh tomee:8-jre-1.7.5-webprofile
testing tomee:8-jre-1.7.5-webprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.0.4-plume
Using bashbrew/cache:4392dd9be6ea2d7b5382d6dae1e739e839a80ad60e6a435522c26d1988a84fe4 (tomee:8-jre-7.0.4-plume)
Tagging tomee:8-jre-7.0.4-plume

$ test/run.sh tomee:8-jre-7.0.4-plume
testing tomee:8-jre-7.0.4-plume
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.0.4-plus
Using bashbrew/cache:c3ad690186e386c653a96b981efbb6e9990deb724c21f1b937d5a45e0d960a36 (tomee:8-jre-7.0.4-plus)
Tagging tomee:8-jre-7.0.4-plus

$ test/run.sh tomee:8-jre-7.0.4-plus
testing tomee:8-jre-7.0.4-plus
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.0.4-webprofile
Using bashbrew/cache:de71022c08edeecb2e651ea4e1e877363e32b67410977fe0fd12a088ad3e04e8 (tomee:8-jre-7.0.4-webprofile)
Tagging tomee:8-jre-7.0.4-webprofile

$ test/run.sh tomee:8-jre-7.0.4-webprofile
testing tomee:8-jre-7.0.4-webprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.0.5-plume
Using bashbrew/cache:5aab8855a255b7504c6e0e965b706b65797e4611b540c63352de65e94a566716 (tomee:8-jre-7.0.5-plume)
Tagging tomee:8-jre-7.0.5-plume

$ test/run.sh tomee:8-jre-7.0.5-plume
testing tomee:8-jre-7.0.5-plume
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.0.5-plus
Using bashbrew/cache:55d1f4bb6bc7facdd0e7acfcd723745603f9c5a54d8979d7630d0d6a09856122 (tomee:8-jre-7.0.5-plus)
Tagging tomee:8-jre-7.0.5-plus

$ test/run.sh tomee:8-jre-7.0.5-plus
testing tomee:8-jre-7.0.5-plus
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.0.5-webprofile
Using bashbrew/cache:3a5c23bcc372b8aa5626066975f546408f0aacf69b7273ea1a48ef4e6c1d346a (tomee:8-jre-7.0.5-webprofile)
Tagging tomee:8-jre-7.0.5-webprofile

$ test/run.sh tomee:8-jre-7.0.5-webprofile
testing tomee:8-jre-7.0.5-webprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.1.0-plume
Using bashbrew/cache:0b47be92ad89716c15a51b943a509cbb69621e781d889bc49db34c19c29d77e4 (tomee:8-jre-7.1.0-plume)
Tagging tomee:8-jre-7.1.0-plume

$ test/run.sh tomee:8-jre-7.1.0-plume
testing tomee:8-jre-7.1.0-plume
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.1.0-plus
Using bashbrew/cache:66d26414c8d56280341c2bb4a44a5f3f85ffab02d34bcc14cd2f3afc6b1da876 (tomee:8-jre-7.1.0-plus)
Tagging tomee:8-jre-7.1.0-plus

$ test/run.sh tomee:8-jre-7.1.0-plus
testing tomee:8-jre-7.1.0-plus
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.1.0-microprofile
Using bashbrew/cache:ccb4ff6e26c07ee62ca13e384f00cd5688319cd4a84ca166c0adc6486e708301 (tomee:8-jre-7.1.0-microprofile)
Tagging tomee:8-jre-7.1.0-microprofile

$ test/run.sh tomee:8-jre-7.1.0-microprofile
testing tomee:8-jre-7.1.0-microprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:8-jre-7.1.0-webprofile
Using bashbrew/cache:d11b7743b1f5e73b7fa0e37dbff3a76dfa446b8688c61ca79ebf3c704062142f (tomee:8-jre-7.1.0-webprofile)
Tagging tomee:8-jre-7.1.0-webprofile

$ test/run.sh tomee:8-jre-7.1.0-webprofile
testing tomee:8-jre-7.1.0-webprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:11-jre-8.0.0-M3-plume
Using bashbrew/cache:9e8397a58ed3e01874fa9bba94a8da624c7d39298eabb26ee8a12dfb0885ee40 (tomee:11-jre-8.0.0-M3-plume)
Tagging tomee:11-jre-8.0.0-M3-plume

$ test/run.sh tomee:11-jre-8.0.0-M3-plume
testing tomee:11-jre-8.0.0-M3-plume
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:11-jre-8.0.0-M3-plus
Using bashbrew/cache:ca405172e872830c2f73263c1cdf1baae03b3af51d762f671999909ee37bec6d (tomee:11-jre-8.0.0-M3-plus)
Tagging tomee:11-jre-8.0.0-M3-plus

$ test/run.sh tomee:11-jre-8.0.0-M3-plus
testing tomee:11-jre-8.0.0-M3-plus
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:11-jre-8.0.0-M3-microprofile
Using bashbrew/cache:6593c69b941adb7e5b928d41c42208094139c165412eb9ff1f55f9a858032580 (tomee:11-jre-8.0.0-M3-microprofile)
Tagging tomee:11-jre-8.0.0-M3-microprofile

$ test/run.sh tomee:11-jre-8.0.0-M3-microprofile
testing tomee:11-jre-8.0.0-M3-microprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build tomee:11-jre-8.0.0-M3-webprofile
Using bashbrew/cache:f7bf23f4ada0eb40667968417826ecf63918614e8cc5aea4bb22e383e9cd9528 (tomee:11-jre-8.0.0-M3-webprofile)
Tagging tomee:11-jre-8.0.0-M3-webprofile
Tagging tomee:latest

$ test/run.sh tomee:11-jre-8.0.0-M3-webprofile
testing tomee:11-jre-8.0.0-M3-webprofile
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Comment by jgallimore** [comment]
That's wonderful, thank you!

**Pull request merged**